### PR TITLE
fix messaging event detail

### PR DESCRIPTION
**Pull request by snopoke**
## Summary
The detail endpoint is failing because of the missing `date_last_activity` field. This updates the detail endpoint to use the same base query as the list endpoint to ensure compatibility.

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Automated test coverage
added

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Comment by snopoke** [comment]
removed high risk label since this is limited to the new messaging event API

File: corehq/apps/api/tests/test_messaging_event_api.py [hunk 1]
```diff
@@ -600,3 +583,33 @@ def _create_sms_messages(domain, count, randomize):
...
+                'phone_number': '99912345678',
+                'content': 'test sms text',
+                'date': '2016-01-01T12:00:00',
+                'date_modified': sms.date_modified.isoformat() if sms.date_modified else None,
+                'direction': 'outgoing',
+                'status': 'sent',
+                'type': 'sms'
+            }
+        ],
+        # "id": 1,  # ids are explicitly removed from comparison
```
Comment (orangejenny) [inline corehq/apps/api/tests/test_messaging_event_api.py:611]: should this be deleted now that `id` is included?

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/api/tests/test_messaging_event_api.py (hunk 1 above)
Comment (snopoke) [reply corehq/apps/api/tests/test_messaging_event_api.py:611]: ah, yes, thanks
```suggestion
```

**Comment by snopoke** [comment]
Build passed on last 'real' commit, merging.

**Pull request merged**